app/views/posts/show.html.erb

```diff
@@ -26,4 +26,7 @@
         </div>
     <% end %> 
   </div>
+  <%= render partial: "likes/like", locals: { post: @post, user: @user } %>
+  <h3>Add new Comment:</h3>
+  <%= render partial: "comments/comment", locals: { post: @post, user: @user } %>
```

Comment:
- [ ] Instead of here on the posts show page, there should be a form in the `views/comments/new.html.erb` file which accepts input for creating comments according to project requirements 👍

Reply:
@codepantha Isn't that just renaming the file, and the result is the same...? Also it is not a requirement to have it on a separate page. This was it makes sense